.golangci.yml

```diff
@@ -60,7 +60,10 @@ linters-settings:
     settings:
       mnd:
         # don't include the "operation", "argument" and "assign"
-        checks: case,condition,return
+        checks: 
+          - case
+          - condition
+          - return
```

Comment:
Is this configuration valid? It seems that not appear in the official gomnd [docs](https://golangci-lint.run/usage/linters/#gomnd)

Reply:
![image](https://user-images.githubusercontent.com/57396775/151512221-8e01c1ae-3207-48d7-9bc7-24fecb629d9d.png)

Reply:
https://github.com/golangci/golangci-lint/pull/2498/files

Reply:
ok, got it